packages/util/src/Pseudonyms.ts

```diff
@@ -0,0 +1,55 @@
+const PSEUDONYMS = [
```

Comment:
lol

Reply:
Not a blocker but an idea in case it's useful: If we need a bigger list to avoid collisions, I've seen fun systems which do a combinatorial combination of adjective and animal to get a unique identifier.  We used to use them to name severe incidents in prod.

Reply:
i was looking into this fun [animal hash](https://github.com/helium/angry-purple-tiger) that we could move to if we needed that. preserving the custom font would be a bit annoying but we could just use a character mapping. adjective + animal would be pretty simple too - will move to this if collisions start becoming more common